frontend/src/libs/components/input/input.tsx

```diff
@@ -30,7 +36,7 @@ const Input = <T extends FieldValues>({
   isDisabled,
 }: Properties<T>): JSX.Element => {
   const { field } = useFormController({ name, control });
-
+  const [showPassword, setShowPassword] = useState(false);
```

Comment:
It is better to call boolean using be/have verbs. For example: `isPasswordShown`.